vendor/vendor.json

```diff
@@ -39,6 +39,12 @@
 			"revision": "c2828203cd70a50dcccfb2761f8b1f8ceef9a8e9",
 			"revisionTime": "2018-01-10T05:33:47Z"
 		},
+		{
```

Comment:
Why did you add this section?

Reply:
This is auto-generated by govendor which a standard tool to import packages into project.

Reply:
got it, but its revisionTime is a little strange.